Gems/AWSCore/Code/Include/Framework/HttpClientComponent.h

```diff
@@ -1,69 +0,0 @@
-/*
```

Comment:
While this is currently unused by code its left in as convenience to implementors. Not 100% convinced brute force removal of any code that is not in use is the right thing todo.

Reply:
The search was done with brute force, not the removal. 
I did do a pass on each case. In this particular case, this header has functions that are not implemented, so as is, is not possible to use it. Including and trying to use `HttpClientComponent` would give linking issues because of missing function implementations of:
- `HttpClientComponent::Init`
- `HttpClientComponent::Activate`
- `HttpClientComponent::Deactivate`
- `HttpClientComponent::MakeHttpRequest`
- `HttpClientComponent::Reflect`
In this particular case, I would argue having a header that is unusable in a public include folder is not a good thing. 

In some cases, unused code was being included by the build (e.g. this header may be included by some compilation unit) which adds to parsing and compilation time.

Reply:
Thanks for being thorough, I thought this code was more complete (should have checked).